r/vignettes/developing.Rmd

```diff
@@ -1,21 +1,23 @@
 ---
 title: "Arrow R Developer Guide"
-output: rmarkdown::html_vignette
+output: 
+  html_document:
+      toc: true
+      toc_depth: 2
 vignette: >
   %\VignetteIndexEntry{Arrow R Developer Guide}
   %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
+
```

Comment:
Super nit picky: is this new line needed?

Reply:
I'll remove